lib/llvm/ReadWriteGraph/Calls.cpp

```diff
@@ -111,12 +111,10 @@ LLVMReadWriteGraphBuilder::createUnknownCall(const llvm::CallInst *CInst) {
 
     // every pointer we pass into the undefined call may be defined
     // in the function
-    for (unsigned int i = 0; i < CInst->getNumArgOperands(); ++i) {
-        const Value *llvmOp = CInst->getArgOperand(i);
-
+    for (const auto &arg : llvmutils::args(CInst)) {
```

Comment:
Is that necessary to take `arg` by reference? Isn't it a pointer?

Reply:
No, the range returns a reference to `llvm::Use`. Other possibility is to use `const llvm::Value *arg` as `Use` can be implicitly converted to `Value`. Which one do you prefer?

Reply:
Oh, the implicit conversion is what puzzled me later in the code. In that case, leave the reference there.